help/en/releasenotes/current-draft-note.shtml

```diff
@@ -383,7 +383,7 @@
 
         <h4>ZIMO</h4>
             <ul>
-                <li>small correction to the files for FW v39 and v40</li>
```

Comment:
Is this line intended to be removed?

Reply:
Oh sorry. I forgot that this change is not included in the current release 5.3.6. Please change to:

        <h4>ZIMO</h4>
            <ul>
                <li>small correction to the files for FW v39 and v40 and added MN Non-Sound Decoders</li>

Reply:
I have fixed it now.